exercises/04_count_rune/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+)
+
+func CountRune(s string, b rune) int {
+	var count int
+	s1 := []rune(s)
+	for i := 0; i < len(s); i++ {
```

Comment:
Your program should be failing for any unicode examples since `len(s)` is longer than `len([]rune(s))` for strings that contain unicode characters

Reply:
Yup, just confirmed that by running your tests